imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerConfig.java

```diff
@@ -24,6 +25,9 @@
     private boolean showCamera;
     private boolean returnAfterCapture;
 
+    private int actionBarColor;
```

Comment:
I think we should add `@ColorInt` in this field and setter getter for documentation and more _linting_ purpose.